src/main/java/org/commonwl/view/workflow/QueuedWorkflow.java

```diff
@@ -5,10 +5,12 @@
 import org.commonwl.view.cwl.CWLToolStatus;
 import org.springframework.data.annotation.Id;
 
+import java.util.List;
+
 /**
  * A workflow pending completion of cwltool
  */
-@JsonIgnoreProperties(value = {"id", "tempRepresentation"})
+@JsonIgnoreProperties(value = {"id", "tempRepresentation", "workflowList"})
```

Comment:
The JSON API never "magically" created workflows through `GET` calls, only by sending a `POST` to `/workflows`. I have used the following for multiple packed workflows:

```
HTTP/1.1 422 Unprocessable Entity
{
    "message": "This workflow file is packed and contains multiple workflow descriptions. Please provide a packedId parameter with one of the following values"
    "packedId": [
        "workflow.cwl",
        "main",
        "workflow_exome.cwl",
        "align.cwl",
        "workflow.cwl_2"
    ]
}
```

And parameterised the `packedWorkflow` parameter when posting similar to branch and path.